nuntium/tests/moderation_messages_test.py

```diff
@@ -201,7 +201,7 @@ def test_message_resend_form(self):
         self.assertEquals(outbound_message_to_pedro.status, 'sent')
 
         data = {
-            'contact_ids': [outbound_message_to_pedro.contact.pk],
+            'person_ids': [outbound_message_to_pedro.contact.pk],
```

Comment:
@desafinadude this should be sending `outbound_message_to_pedro.contact.person.pk`